2024-12-29 19:50:01,006 ERROR scheduler Exception in Enqueue Events for Site employe.test
Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 61, in wrapper
    return _cache[func][args_key]
           ~~~~~~~~~~~~^^^^^^^^^^
KeyError: 5740354900026072187

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1560, in _load_app_hooks
    app_hooks = get_module(f"{app}.hooks")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1426, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'employe_management'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 108, in enqueue_events_for_site
    enqueue_events()
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in enqueue_events
    all_jobs = frappe.get_all("Scheduled Job Type", filters={"stopped": 0}, fields="*")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 2015, in get_all
    return get_list(doctype, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1990, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 203, in execute
    result = self.build_and_run()
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 218, in build_and_run
    args = self.prepare_args()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 258, in prepare_args
    self.build_conditions()
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 566, in build_conditions
    self.build_filter_conditions(self.filters, self.conditions)
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 581, in build_filter_conditions
    conditions.append(self.prepare_filter_condition(f))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 692, in prepare_filter_condition
    additional_filters_config = get_additional_filters_from_hooks()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/boot.py", line 430, in get_additional_filters_from_hooks
    filter_hooks = frappe.get_hooks("filters_config")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1591, in get_hooks
    hooks = _dict(_load_app_hooks())
                  ^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 63, in wrapper
    return_val = func(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1566, in _load_app_hooks
    print(f'Could not find app "{app}": \n{e}')
BrokenPipeError: [Errno 32] Broken pipe
2024-12-29 19:54:00,020 ERROR scheduler Exception in Enqueue Events for Site employe.test
Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 61, in wrapper
    return _cache[func][args_key]
           ~~~~~~~~~~~~^^^^^^^^^^
KeyError: 5740354900026072187

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1560, in _load_app_hooks
    app_hooks = get_module(f"{app}.hooks")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1426, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'employe_management'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 108, in enqueue_events_for_site
    enqueue_events()
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in enqueue_events
    all_jobs = frappe.get_all("Scheduled Job Type", filters={"stopped": 0}, fields="*")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 2015, in get_all
    return get_list(doctype, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1990, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 203, in execute
    result = self.build_and_run()
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 218, in build_and_run
    args = self.prepare_args()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 258, in prepare_args
    self.build_conditions()
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 566, in build_conditions
    self.build_filter_conditions(self.filters, self.conditions)
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 581, in build_filter_conditions
    conditions.append(self.prepare_filter_condition(f))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 692, in prepare_filter_condition
    additional_filters_config = get_additional_filters_from_hooks()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/boot.py", line 430, in get_additional_filters_from_hooks
    filter_hooks = frappe.get_hooks("filters_config")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1591, in get_hooks
    hooks = _dict(_load_app_hooks())
                  ^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 63, in wrapper
    return_val = func(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1566, in _load_app_hooks
    print(f'Could not find app "{app}": \n{e}')
BrokenPipeError: [Errno 32] Broken pipe
2024-12-29 19:58:00,031 ERROR scheduler Exception in Enqueue Events for Site employe.test
Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 61, in wrapper
    return _cache[func][args_key]
           ~~~~~~~~~~~~^^^^^^^^^^
KeyError: 5740354900026072187

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1560, in _load_app_hooks
    app_hooks = get_module(f"{app}.hooks")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1426, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'employe_management'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 108, in enqueue_events_for_site
    enqueue_events()
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in enqueue_events
    all_jobs = frappe.get_all("Scheduled Job Type", filters={"stopped": 0}, fields="*")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 2015, in get_all
    return get_list(doctype, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1990, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 203, in execute
    result = self.build_and_run()
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 218, in build_and_run
    args = self.prepare_args()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 258, in prepare_args
    self.build_conditions()
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 566, in build_conditions
    self.build_filter_conditions(self.filters, self.conditions)
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 581, in build_filter_conditions
    conditions.append(self.prepare_filter_condition(f))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 692, in prepare_filter_condition
    additional_filters_config = get_additional_filters_from_hooks()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/boot.py", line 430, in get_additional_filters_from_hooks
    filter_hooks = frappe.get_hooks("filters_config")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1591, in get_hooks
    hooks = _dict(_load_app_hooks())
                  ^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 63, in wrapper
    return_val = func(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1566, in _load_app_hooks
    print(f'Could not find app "{app}": \n{e}')
BrokenPipeError: [Errno 32] Broken pipe
2024-12-29 20:02:00,042 ERROR scheduler Exception in Enqueue Events for Site employe.test
Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 61, in wrapper
    return _cache[func][args_key]
           ~~~~~~~~~~~~^^^^^^^^^^
KeyError: 5740354900026072187

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1560, in _load_app_hooks
    app_hooks = get_module(f"{app}.hooks")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1426, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'employe_management'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 108, in enqueue_events_for_site
    enqueue_events()
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in enqueue_events
    all_jobs = frappe.get_all("Scheduled Job Type", filters={"stopped": 0}, fields="*")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 2015, in get_all
    return get_list(doctype, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1990, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 203, in execute
    result = self.build_and_run()
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 218, in build_and_run
    args = self.prepare_args()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 258, in prepare_args
    self.build_conditions()
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 566, in build_conditions
    self.build_filter_conditions(self.filters, self.conditions)
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 581, in build_filter_conditions
    conditions.append(self.prepare_filter_condition(f))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/model/db_query.py", line 692, in prepare_filter_condition
    additional_filters_config = get_additional_filters_from_hooks()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/boot.py", line 430, in get_additional_filters_from_hooks
    filter_hooks = frappe.get_hooks("filters_config")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1591, in get_hooks
    hooks = _dict(_load_app_hooks())
                  ^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/utils/caching.py", line 63, in wrapper
    return_val = func(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^
  File "/home/hardik/employe-bench/apps/frappe/frappe/__init__.py", line 1566, in _load_app_hooks
    print(f'Could not find app "{app}": \n{e}')
BrokenPipeError: [Errno 32] Broken pipe
